stainless_extraction/src/lib.rs

```diff
@@ -131,6 +131,19 @@ struct BaseExtractor<'l, 'tcx: 'l> {
   extraction: Option<Box<Extraction<'l>>>,
 }
 
+fn sanitize_path_name(s: &String) -> String {
+  // Remove forbidden characters
+  let s = s.replace(&[' ', '<', '>'][..], "");
```

Comment:
Can we add a check (eg. `assert!`, don't think it's worth to do proper error handling there) for the case where `s`is empty after the replacement? Not that I expect this could actually happen, but we never know and if we don't catch it here, it might be difficult to figure out what went wrong later down the pipeline.